runtime/precompiles/src/staking.rs

```diff
@@ -54,6 +54,7 @@ where
 		// Parse the function selector
 		let inner_call = match input[0..SELECTOR_SIZE_BYTES] {
 			[0x82, 0xf2, 0xc8, 0xdf] => Self::nominate(&input[SELECTOR_SIZE_BYTES..])?,
+			[0xad, 0x76, 0xed, 0x5a] => Self::join_candidates(&input[SELECTOR_SIZE_BYTES..])?,
```

Comment:
The call_data is hardcoded? Where did these bytes come from?

Would it make sense to make it the first four bytes of the keccak-256 of the method name or some other standard?

Reply:
Never mind, I found your comment with this link https://ethereum.stackexchange.com/a/73405/9963